lib/object_store/app/controllers/object_store/content_blocks_controller.rb

```diff
@@ -1,5 +1,58 @@
 class ObjectStore::ContentBlocksController < ApplicationController
+  include ObjectStore::Engine.routes.url_helpers
+
+  before_action :set_content_block, only: %i[show update destroy]
+
+  def info
+    block_type = params.require(:block_type)
+    @schema = ObjectStore::ContentBlockValidator.schema_for(block_type)
```

Comment:
I'm wondering whether we should break up the fetching of the schema from the Validator itself? It feels a bit weird to fetch a schema from a validator - then we can call the Schema fetching service within the validator when validating